src/posh-git.psm1

```diff
@@ -78,6 +78,7 @@ $GitPromptScriptBlock = {
 
     # This has to be *after* the call to Write-VcsStatus, which populates $global:GitStatus
     Set-WindowTitle $global:GitStatus $IsAdmin
+    Set-TabTitle $global:GitStatus
```

Comment:
Even if folks don't use $IsAdmin in their customized scriptblock, I think we should pass it.  No harm in having this extra bit of info injected into the scriptblock.